modules/ROOT/pages/8.2.0-release-notes.adoc

```diff
@@ -42,6 +42,23 @@ The following premium plugin updates were released alongside {productname} 8.2.0
 
 // For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].
 
+=== Suggested Edits
+
+The {productname} {release-version} release includes an accompanying release of the **Suggested Edits** premium plugin.
+
+**Suggested Edits** includes the following improvement.
+
+=== Operation descriptions now shows the type of content that was changed.
```

Comment:
```suggestion
=== Suggestions now display the type of content that was changed.
```

Reply:
Happy to update to reflect this change, however we should also update `suggestededits-TINY-12852-2025-10-02.yaml` also.

Reply:
In the code we call them operations, so that's still fine.